en/docs/develop/customizations/customizing-the-developer-portal/customize-developer-portal-credentials.md

```diff
@@ -0,0 +1,103 @@
+# Customize Developer Portal Credential Preferences
+  
+You can change your WSO2 API Manager Developer Portal password, customize the default Developer Portal password policy, and enable or disable the Developer Portal password policy guidelines based on your requirement.
+
+## Change the Developer Portal password
+
+Follow the instructions below to change your password:
+
+1. Sign in to the Developer Portal.
+
+     `https://<hostname>:9443/devportal`
+
+     Example: `https://localhost:9443/devportal`
+
+     Use your username and password to sign in
+  
+2. Click on your username that appears at the top right corner and select **Change Password**.
+  
+     <img src="{{base_path}}/assets/img/learn/change-devportal-password-user-menu-click.png" alt="Change Developer Portal password User Menu" width="300px"/>
```

Comment:
why are we using HTML?

Reply:
This is because it is straightforward than having to resize the image as there is no way to add the image width in mkdocs as in all the options I have checked didnt work.